lncfg/address.go

```diff
@@ -245,12 +245,15 @@ func ParseAddressString(strAddress string, defaultPort string,
 		// identifiable error.
 		addr, err := tcpResolver("tcp", addrWithPort)
 		if err != nil {
-			torErrStr := "tor host is unreachable"
-			if strings.Contains(err.Error(), torErrStr) {
+			switch err.Error() {
+			case "tor host is unreachable",
+				"tor general error",
```

Comment:
From what I've seen in user logs, this can arise if a user attempts to connect out after tor has been restarted, but before `lnd` has re-initialized with the control port